operator-framework/src/test/java/io/javaoperatorsdk/operator/processing/retry/GenericRetryExecutionTest.java

```diff
@@ -20,13 +18,13 @@ public void delayIsMultipliedEveryNextDelayCall() {
         RetryExecution retryExecution = getDefaultRetryExecution();
 
         Optional<Long> res = callNextDelayNTimes(retryExecution, 2);
-        assertThat(res.get()).isEqualTo(DEFAULT_INITIAL_INTERVAL);
```

Comment:
This files consist undesired changes related to auto refactoring code using IDEA, but it looks good.